src/account/account-updated.event.ts

```diff
@@ -0,0 +1,13 @@
+import type { Account } from './account.entity';
+
+export class AccountUpdatedEvent {
+    static getName(): string {
+        return 'account.updated.event';
```

Comment:
This could just be `account.updated`, but I think it would contradict the current event. Maybe something to clean up later?